lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -824,7 +824,8 @@ sub investigate {
     my $investigation = $job->investigate;
     $self->respond_to(
         json => {json => $investigation},
-        html => {text => join("\n", map { "$_: $investigation->{$_}" } keys %$investigation)});
+        html => {inline => join("\n", map { "<tr><td style=\"vertical-align: top;\"><b>$_:</b></td><td><pre>$investigation->{$_}</pre></td><tr>" } keys %$investigation)}
```

Comment:
The keys should be sorted.

Reply:
of course, agreed. I had this change but couldn't make it before the christmas party ;) Updated